<ui version="4.0" >
 <author>Adriaan de Groot</author>
 <class>VCalWidget</class>
 <widget class="QWidget" name="VCalWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>209</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>5</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize" >
   <size>
    <width>570</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Calendar-Conduit Options</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="Widget2" >
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="2" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" >
        <widget class="Q3ButtonGroup" name="fSyncDestination" >
         <property name="title" >
          <string>Sync Destination</string>
         </property>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" colspan="2" >
           <widget class="QRadioButton" name="fSyncStdCalendar" >
            <property name="whatsThis" >
             <string>&lt;qt>Select this option to synchronize with the calendar specified by the KDE calendar settings.&lt;/qt></string>
            </property>
            <property name="text" >
             <string>&amp;Standard calendar</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QRadioButton" name="fSyncFile" >
            <property name="enabled" >
             <bool>true</bool>
            </property>
            <property name="whatsThis" >
             <string>&lt;qt>Select this option to use a specific calendar file, instead of the standard KDE calendar. This file must be in the in the vCalendar or iCalendar format. Enter the location of this file in the edit box or select it clicking the file picker button.&lt;/qt></string>
            </property>
            <property name="text" >
             <string>Calendar &amp;file:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="KUrlRequester" native="1" name="fCalendarFile" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="sizePolicy" >
             <sizepolicy>
              <hsizetype>3</hsizetype>
              <vsizetype>5</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="whatsThis" >
             <string>&lt;qt>Enter here the location and filename of the calendar file or select it clicking the file picker button. This file must be in the iCalendar or vCalendar format.&lt;/qt></string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QCheckBox" name="fArchive" >
         <property name="whatsThis" >
          <string>When this box is checked, archived records will still
be saved in the calendar on the PC.</string>
         </property>
         <property name="text" >
          <string>Store &amp;archived records in the KDE calendar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Conflicts</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QLabel" name="fTextLabel" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>4</hsizetype>
           <vsizetype>5</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Conflict &amp;resolution:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
         <property name="buddy" >
          <cstring>fConflictResolution</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QComboBox" name="fConflictResolution" >
         <property name="whatsThis" >
          <string>&lt;qt>Select in this list how conflicting entries (entries which were edited both on your handheld and on the PC) are resolved. Possibly values are "Use KPilot's Global Setting" to use the settings defined in KPilot HotSync configuration, "Ask User" to let you decide case by case, "Do Nothing" to allow the entries to be different, "PC overrides", "Handheld overrides", "Use values from last sync" and "Use both entries" to create a new entry on both the PC and handheld. Note that this does &lt;i>not&lt;/i> handle double-scheduling conflicts.&lt;/qt></string>
         </property>
         <property name="currentIndex" >
          <number>6</number>
         </property>
         <item>
          <property name="text" >
           <string>Use KPilot's Global Setting</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Ask User</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Do Nothing</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Handheld Overrides</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>PC Overrides</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Values From Last Sync (if possible)</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Use Both Entries</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>fSyncFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>fCalendarFile</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
